README.md

```diff
@@ -21,17 +24,18 @@ New features like a new network concept will be part of future releases.
 * the LAN-interface (br-lan) is not providing a default route on a unconfigured node, so manual reconfiguration is required when not using the assistent
```

Comment:
The lline "in preinit and failsafe it IP-addr..." does not read correctly.  This should probably be reworded.  Maybe something like "the IP-address in pre-init and in failsafe mode is ...."

Reply:
maybe instead of using "preinit", "first boot" or "unconfigured" would be clearer to the user

Reply:
"preinit" is very different from "first boot" and "default"-node